### PR TITLE
table

### PR DESCRIPTION
**Pull request by satya-subham**

### CODE REVIEW

File: src/Components/Layout/StockLayout.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Cards from '../Cards/Cards'
 
 import "../Layout/StockLayout.css";
 import '../Cards/Cards.css'
+import { TableComponent } from "../table/TableComponent";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx]: First external imports occur that is imports from installed libraries, then 1 line space, then all internal imports that is files/components/functions that you have created, then 1 line space and then finally all css imports.

File: src/Components/table/TableComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Table } from 'antd'
+import { useState, useEffect } from 'react'
```
Comment (Souvikdeb2612) [inline src/Components/table/TableComponent.jsx]: already importing from react in line 1.

File: src/Components/table/TableComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+export function TableComponent() {
+  const [url, setUrl] = useState('https://financialmodelingprep.com/api/v3/historical-price-full/AAPL?from=2018-03-12&to=2019-03-12&apikey=a3a0e9abc7f90ef141af43c5ff61df6a')
+  const [data, setData] = useState([]);
+
+  useEffect(()=>{
+  axios
+  .get(url)
+  .then((response)=>{
+    console.log(response.data['historical']);
```
Comment (Souvikdeb2612) [inline src/Components/table/TableComponent.jsx]: remove console statements before raising pr

File: src/Components/table/TableComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    setData([...response.data['historical']])
+  })
+  .catch((err)=>{});
+  },[])
+
+  return (
+    <>
+    <Table
+    dataSource={data}
+    columns={[
```
Comment (Souvikdeb2612) [inline src/Components/table/TableComponent.jsx]: static data should be kept in separate files and import here when required.

File: src/Components/table/TableComponent.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      {
+        title: 'Open',
+        dataIndex:'open'
+      }
+    ]}
+    ></Table>
+
+    {
+      data.map((item)=>{
+        console.log(item);
```
Comment (Souvikdeb2612) [inline src/Components/table/TableComponent.jsx]: remove console statements before raising pr

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/table/TableComponent.jsx (hunk 4 above)
Comment (satya-subham) [reply src/Components/table/TableComponent.jsx]: these are component of antd library. how can i export and import ??

File: src/Components/Layout/StockLayout.jsx [hunk 6]
```diff
@@ -2,6 +2,7 @@ import { Layout } from "antd";
 
 import React from "react";
 import Cards from '../Cards/Cards'
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx:4]: 1 line space before starting internal imports

File: src/Components/table/TableComponent.jsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  };
+
+
+  
+  useEffect(() => {
+    axios
+      .get(url)
+      .then((response) => {
+        flattenObj(response.data.rates)
+        console.log(response.data.rates);
```
Comment (Souvikdeb2612) [inline src/Components/table/TableComponent.jsx]: remove console statement before raising pr

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Layout/StockLayout.jsx [hunk 8]
```diff
@@ -4,6 +4,11 @@ import React from "react";
...
 import { TableComponent } from "../table/TableComponent";
 
+import Cards from "../Cards/Cards";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx:9]: cards imported twice

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Layout/StockLayout.jsx (hunk 8 above)
Comment (satya-subham) [reply src/Components/Layout/StockLayout.jsx:9]: sir... one for component and another for css. it's not imported by me.

File: src/Components/Layout/StockLayout.jsx [hunk 9]
```diff
@@ -6,6 +6,13 @@ import { TableComponent } from "../table/TableComponent";
 
 import "../Layout/StockLayout.css";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx:7]: css also imported twice.

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Layout/StockLayout.jsx (hunk 9 above)
Comment (satya-subham) [reply src/Components/Layout/StockLayout.jsx:7]: that's costume css sir

Comment (satya-subham) [reply src/Components/Layout/StockLayout.jsx:7]: if i remove that it's effecting on ui

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**